Resources_details.md

```diff
@@ -0,0 +1,51 @@
+### Setup
+
+ * List of projects per Research Program
+ * Use CMS wiki or my.nci.org.au to find name of project coordinator
+ * Information to give to the project coordinator before joining (CI or delegate)
+ * Request to join the project through my.nci.org.au
+ * Determining resources required, e.g. running model, analysing data
+    - Compute resources (KSU)
+    - Storage resources (GB): short term (`/short`), medium term (`/gdata`) and archive (`mdss`)
+ * Use of `nci_account`
+
+[Link to training](???)
+
+### Monitor resource use
+
+ * Review of the tools available to you
+    * nci_account 
+    * <fsystem>_files_report
+    * lquota
+    * nci_monitor
+    * du
+    * ncdu
+    * find
```

Comment:
@paolap The bash commands here are not in the bash basics. And I don't think they should be. I think it's better to introduce them in this training so they are introduced within the context users are most likely to need them.